js/enums/api.js

```diff
@@ -11,6 +11,7 @@ const cmds = keyMirror({
     registerBoundsChange: null,
     registerProtocolHandler: null,
     registerActivityDetection: null,
+    showAlertSettings: null,
```

Comment:
Can we change the use of alert to notifications?

@lneir I have observed that in some places we use "Alerts" and in some places we use "Notifications". I think we should stick with either one of them for consistency sakes.